### DISCUSSION TITLE
Heading issues in landscape mode

### DISCUSSION THREAD

**Discussion by EFigueira1511** [joaomlourenco/novathesis#495] [Q&A]
Hello, it's me again, João :)

I wanted to have an horizontal page for a figure, so I applied the following code:

\begin{landscape}
\centering
\vspace*{\fill}
\begin{figure}[htpb]
  \centering
  \includegraphics[width=0.9\linewidth]{Chapters/Figures/xxx.png}
  \caption{xxxx}
  \label{fig:xxx}
\end{figure}
\end{landscape}

However, the heading that usually appears in vertical pages remains vertical and the title doesn't show. Could you help me, please?
<img width="816" alt="Screenshot 2024-03-25 at 22 05 28" src="https://github.com/joaomlourenco/novathesis/assets/164951796/192b4cc4-9317-40fa-8af2-1b70df9dde86">

**Comment by joaomlourenco** [comment]
Try instead:
```latex
\begin{newpdflayout}{210mm}{297mm}
    your contentent here
\end{newpdflayout}
```

Let me know if it works for you.

**Reply by EFigueira1511** [reply]
The heading is now appearing horizontally, including the page number, but the chapter title is not displaying in the header as it does on other pages.

<img width="1036" alt="Screenshot 2024-03-27 at 21 49 12" src="https://github.com/joaomlourenco/novathesis/assets/164951796/42e89f8c-b1c7-4ee3-93f7-f64ef6b7b1f0">

**Reply by EFigueira1511** [reply]
Furthermore, after applying the code you suggested, the page following the horizontal page is not functioning as usual. It breaks the page in the middle, as can be observed in the following screenshot. Could you please help me?
<img width="396" alt="Screenshot 2024-03-27 at 22 45 02" src="https://github.com/joaomlourenco/novathesis/assets/164951796/16efe7a6-6828-453d-b45b-69a611d74f41">

**Reply by joaomlourenco** [reply]
It seems you found some bug(s).  I'll check it out…

**Comment by joaomlourenco** [comment]
Both problems were fixed with the new `novathesis.cls`.  Just download it and replace yours.
Let me know if it worked for you.

**Reply by EFigueira1511** [reply]
The first issue regarding the heading has been fixed, thank you. The following page no longer breaks in the middle, but the page number remains centered on the page.

![latex_bug](https://github.com/joaomlourenco/novathesis/assets/164951796/f0b65229-9383-4c55-9bed-412c5052ce21)

**Comment by joaomlourenco** [comment]
Please try with the new `novathesis.cls` file.  Let me know if it worked this time.

**Reply by EFigueira1511** [reply]
It worked, thank you very much!
